scene/animation/tween.cpp

```diff
@@ -560,12 +560,16 @@ void Tween::_tween_process(float p_delta) {
 
 		switch (data.type) {
 			case INTER_PROPERTY:
-			case INTER_METHOD:
-				break;
+			case INTER_METHOD: {
+				Variant result = _run_equation(data);
+				emit_signal("tween_step", object, data.key, data.elapsed, result);
```

Comment:
Wouldn't putting the `emit_signal` here mean that no `tween_step` would be emitted for `INTER_CALLBACK`?

Reply:
Correct. It's a little weird as it's basically acting like a timer when used with `INTER_CALLBACK` types but it works this way in the original code and I tried to keep the semantics the same (minus the `tween_completed` bug). Since there's no actual value being interpolated there's no value to give to emit with `tween_step`.

Reply:
Right... my bad 😃